README.md

```diff
@@ -26,23 +26,32 @@ cellular networks.
 
 ## Installation
 
-You can run this example
-[in the Leap IDE](https://ide.dwavesys.io/#https://github.com/dwave-examples/cooperative-multipoint).
+You can run this example without installation in
+[the Leap IDE](https://ide.dwavesys.io/#https://github.com/dwave-examples/cooperative-multipoint)
+or other cloud-based IDEs that support the 
+[Development Containers specification](https://containers.dev/supporting)
+(aka "devcontainers") such as [GitHub Codespaces](https://docs.github.com/codespaces).
```

Comment:
How about using a start link for Codespaces, like you do for Leap IDE?
```suggestion
(aka "devcontainers") such as [GitHub Codespaces](https://codespaces.new/dwave-examples/coordinated-multipoint-notebook).
```
There's also a "quickstart" option, if you prefer that:
```suggestion
(aka "devcontainers") such as [GitHub Codespaces](https://codespaces.new/dwave-examples/coordinated-multipoint-notebook?quickstart=1).
```

Reply:
For me these will work nicely, but what happens for a user without a GitHub account? What do they see?

Reply:
I guess they get a login/sign-up page for GitHub.